programs/pkey/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+set(libs
+    ${mbedtls_target}
+)
+
```

Comment:
This still is not correct. All that `set(libs $mbedtls_target)` does here is set a variable, which is then not used. Either correct the` target_link_libraries()` call to include `${libs} `rather than `${mbedtls_target}` or remove the setting of `${libs}`